ipinfo_db/client.py

```diff
@@ -43,7 +43,27 @@ def __init__(self, access_token=None, path=None, replace=False):
             urllib.request.urlretrieve(DB_DOWNLOAD_URL+self.access_token, self.path)
 
         # Read the mmdb file.
-        self.db = maxminddb.open_database(self.path)
+        self.db = Reader(self.path)
+
+    def open(self, path):
+        '''Opens the mmdb file located at the given path.
+
+        :param path: Path to mmdb file.
+        '''
+        self.db.open(path)
+
+    def close(self):
```

Comment:
We can leave this one here though, for a clear path to cleaning up the instance for advanced users.